docs/faqs/faqs.md

```diff
@@ -23,14 +23,15 @@ In order to deploy the contract on the public Ethereum network, you'll need the
 - A deployment script (two of the most popular deployment tools are [Hardhat](https://hardhat.org/guides/deploying.html) and [Truffle](https://trufflesuite.com/docs/truffle/advanced/networks-and-app-deployment/))
 - Access to a node on the network
 
-Once your contract is deployed, you can view the contract details through popular blockchain explorers such as [Etherscan](https://etherscan.io/), which will show all transaction on the public ethereum network interactions with the contract and show any token value associated with the contract. It's worth noting that Etherscan is a block explorer for public Ethereum, so if you deploy on a different network you'll need to find a block explorer that services that network specifically.
-
-Kaleido has built in Smart Contract deployment and management functionality.  See our [documentation](https://docs.kaleido.io/kaleido-services/smart-contract-management/usage/) here to learn more about how you can deploy smart contracts to your private network hosted on Kaleido.
+Kaleido has built in Smart Contract deployment and management functionality. Additionally, you can you use the [Kaleido Truffle box](https://docs.kaleido.io/developers/smart-contracts/truffle/) to help you handle the management, deployment, and testing of smart contracts. See our [documentation](https://docs.kaleido.io/kaleido-services/smart-contract-management/usage/) here to learn more about how you can deploy smart contracts to your private network hosted on Kaleido. 
 
 For additional information about Smart Contracts, please see the official [Ethereum.org documentation](https://ethereum.org/en/developers/docs/smart-contracts/)
 
 ## 🦊 Can I connect FireFly to MetaMask?
 Yes! Before you set up MetaMask you'll likely want to create some tokens that you can use to send between wallets on your FF network. Go to the tokens tab in your FireFly node's UI, create a token pool, and then mint some tokens. Once you've done this, follow the steps listed [here](https://hyperledger.github.io/firefly/tutorials/tokens/erc721.html#use-metamask) to set up MetaMask on your network.
 
+## Can I connect FireFly to an existing Fabric Network
+
```

Comment:
Should this have content in it? Or should we wait to add the header until we have the content ready?